src/components/work/details/utils/details.utils.js

```diff
@@ -975,6 +975,7 @@ export function filterAndMerge({ baseArray, extendingArray }) {
  * @param {object} manifestation
  */
 export const createEditionText = (manifestation) => {
+  if (!manifestation) return "";
```

Comment:
Den her rettelse lavede du hvorfor? For det var `createEditionText` jeg havde problemer med, hvor det hjalp at kigge på `materialId` og `materialTypes` i stedet for `bookmarkId`

Reply:
min kode crashede efter en ændring jeg havde lavet med en manfestation var undefined fejl. derfor vil jeg være defensiv her

Reply:
Ja, men jeg tror det var dte samme problem med manglende BookmarkId, så